### DISCUSSION TITLE
Different button icons

### DISCUSSION THREAD

**Discussion by CyberLykan** [vladmandic/sdnext#1220] [General]
### Issue Description

The buttons in `txt2img_tools` and `img2img_tools` are displaying different icons. Tried default gradio theme and storj theme.

![image](https://github.com/vladmandic/automatic/assets/21145060/dbf20e2f-39ed-481a-94a7-201823e4cf78)

### Version Platform Description

Version: 873371f1 Tue May 30 14:13:29 2023 -0400

Edition	Windows 10 Pro
Version	22H2
Installed on	‎12/‎26/‎2021
OS build	19045.2965
Experience	Windows Feature Experience Pack 1000.19041.1000.0

```
Torch 2.1.0.dev20230530+cu121
Torch backend: nVidia CUDA 12.1 cuDNN 8801
Torch detected GPU: NVIDIA GeForce GTX 1080 VRAM 8192 Arch (6, 1) Cores 20
```

Google Chrome Version 113.0.5672.127 (Official Build) (64-bit)

```
Extensions enabled: ['a1111-sd-webui-lycoris', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'prompt-bracket-checker', 'ScuNET',
                         'sd-extension-aesthetic-scorer', 'sd-webui-controlnet',
                         'stable-diffusion-webui-images-browser', 'SwinIR', 'sd-webui-stablesr',
                         'sd-webui-supermerger', 'stable-diffusion-webui-model-toolkit',
                         'stable-diffusion-webui-wd14-tagger']
```

**Comment by vladmandic** [comment]
and how do you define "wrong" when icon choice is exactly that - a choice?
i'm converting this into discussion instead of an issue.

**Reply by CyberLykan** [reply]
> and how do you define "wrong" when icon choice is exactly that - a choice? i'm converting this into discussion instead of an issue.

I do not recall ever having the choice to change the icons.

Aggressive tone aside, the change may have been caused by commit https://github.com/vladmandic/automatic/commit/8354b7c6d95c8c447ad7179ed5d7d63e1a02bf34.

This is how they should appear on my system.
![image](https://github.com/vladmandic/automatic/assets/21145060/468f45bb-6299-4579-b753-cbabc82f8518)

**Reply by vladmandic** [reply]
yes, that's correct.
and i'd love some suggestions (not just say you're used to old, that's not good enough of a reason for me).

**Reply by CyberLykan** [reply]
So then why do the new codes show the incorrect icons for me? Do I have to manually change them back in the code to get the correct icons?

**Reply by vladmandic** [reply]
again, why do you think those are incorrect? they are different - that doesn't mean incorrect!
yes, icons have changed. yes, it was my choice. yes, i'm open to suggestions.

**Reply by CyberLykan** [reply]
Fair enough.
My specific issue is that the two style icons don't really imply they're used for styles. The others are fine.

**Reply by lbeltrame** [reply]
I think @CyberLykan didn't notice that the actual Unicode in the code does not match the one in the comment next to it.

**Comment by SoraEnkai** [comment]
I think personally the icons look okay, aside from the `clear prompt` one, as it's currently accented on a person throwing trash.
Below is how icons currently look on nota-ai/theme and the `clear prompt` icon isn't clear what it does until i hover to get the label
![Screenshot 2023-05-30 223300](https://github.com/vladmandic/automatic/assets/39128897/4312c62d-989d-40f7-8abf-6dd5ab3a50cf)

I think something like heroicons trash icon could be used there instead. (not sure what is used in the theme/gradio right now)
![image](https://github.com/vladmandic/automatic/assets/39128897/df2ab9eb-69b0-4d33-939c-8801856b7c4f)

https://heroicons.com/

**Reply by CyberLykan** [reply]
Thanks for your feedback. The two style icons at the end also don't seem to imply they are used for styles.

**Reply by SoraEnkai** [reply]
I haven't really used them, so i can't comment, but the last icon `save styles` also suffers from the same issue, busy icon in a relatively small size. Maybe something like styled save icon, but ideally it should be a single large icon like 1st and 3rd icons. 

This would also help make them be visibly identifiable on different screen sizes.

Using heroicons as example again, something like `arrow-down-on-square`,`arrow-down-tray`, `document-arrow-down` or `bookmark` as icon

**Reply by vladmandic** [reply]
yeah, there are no icons in this ui, its unicode symbols. so if you find unicode character that fits, let me know.
new ui will have actual icons.

**Reply by Wrenched** [reply]
I have poor eyesight. These are small enough on the screen that I have to rely on remembering their position:function match as many of them are squares of the same color with slight variations of "scribbles" inside. Different colors would help. With the old ones I would simply click on "red" for loras, but now I have to remember that it's the third icon. I wouldn't go back to red for "extra networks" personally and would probably use it for "clear prompts", as it seems most fitting for delete/trash, but you get the idea.

Position, shape, color, etc., all have an impact on how people distinguish function, and different people will prioritize different aspects.
